### ISSUE TITLE
Expedition timer was changed

### ISSUE THREAD

**Issue by ElementOfCoffee** [perryhuynh/kcauto#372]
* kcauto version: 7.5.0
* Viewer: Chrome
* OS: Win10

In the last patch, the expedition timer was changed, which led to the inoperability of the entire expedition module. Updating the file **expedition_timer.png** by address **../kcauto/kcauto.sikuli/expedition.sikuli** solves the problem. The updated file is attached below.

![expedition_timer](https://user-images.githubusercontent.com/28761772/62709100-6a222d80-b9fd-11e9-8722-9bb13422d9ab.png)

**Comment by ForsakenRei** [comment]
I cannot send expedition out for fleet 3 and 4 after today's update. But fleet 2 works totally fine.
I think the timer has two different colors? One is darker at the beginning of the expedition, the other is the one attached with a lighter green when the expedition is towards to ending.

**Comment by perryhuynh** [comment]
It's because the fleet icons were changed, for time time being if you want to do expeditions, you can generate your own assets for the following in `kcauto.sikuli/fleet.sikuli`:
```
fleet_1.png
fleet_1_active.png
fleet_2.png
fleet_2_active.png
fleet_3.png
fleet_3_active.png
fleet_4.png
fleet_4_active.png
```
as well as change the following:
https://github.com/mrmin123/kcauto/blob/e52c6ec68a7eb703f4512073c7d97d81f43d0d39/kcauto.sikuli/fleet.sikuli/fleet.py#L81-L82

to use `.similar(0.97)` instead of `.exact()`

**Comment by ForsakenRei** [comment]
> It's because the fleet icons were changed, for time time being if you want to do expeditions, you can generate your own assets for the following in `kcauto.sikuli/fleet.sikuli`:
> 
> ```
> fleet_1.png
> fleet_1_active.png
> fleet_2.png
> fleet_2_active.png
> fleet_3.png
> fleet_3_active.png
> fleet_4.png
> fleet_4_active.png
> ```
> 
> as well as change the following:
> https://github.com/mrmin123/kcauto/blob/e52c6ec68a7eb703f4512073c7d97d81f43d0d39/kcauto.sikuli/fleet.sikuli/fleet.py#L81-L82
> 
> to use `.similar(0.97)` instead of `.exact()`

Yeah thanks. I found those files but not sure which section of code I need change. While it is just a temp fix so it should be fine for today. Will wait for new release.

**Comment by perryhuynh** [comment]
Its in the snippet lol, Line 81 and 82 of `fleet.py`

You change it to
```
 region, Pattern('fleet_{}.png'.format(fleet)).similar(0.97), 
 region, Pattern('fleet_{}_active.png'.format(fleet)).similar(0.97)) 
```

**Comment by ForsakenRei** [comment]
> Its in the snippet lol, Line 81 and 82 of `fleet.py`
> 
> You change it to
> 
> ```
>  region, Pattern('fleet_{}.png'.format(fleet)).similar(0.97), 
>  region, Pattern('fleet_{}_active.png'.format(fleet)).similar(0.97)) 
> ```

I mean...after your last comment I know I need change them lol. Anyway thanks.

**Comment by mrmin123** [comment]
I won't be able to assess the changes until later tonight but I'll take a look then. Thanks for the report and (temporary?) fix.

**Comment by mrmin123** [comment]
Relevant fixes in #373
